CRM/ManualDirectDebit/Upgrader.php

```diff
@@ -10,7 +10,6 @@ class CRM_ManualDirectDebit_Upgrader extends CRM_ManualDirectDebit_Upgrader_Base
   // upgrade tasks. They are executed in order (like Drupal's hook_update_N).
 
   public function install() {
-    $this->executeSqlFile('sql/install.sql');
   }
```

Comment:
can u removed all these unimplemented methods (install, enable, disable and uninstall), you can readd them whenver you want them.